packages/ezflow_extension/ezextension/ezflow/design/standard/templates/page/zone.tpl

```diff
@@ -1,7 +1,11 @@
 <div class="object-left">
     <select name="ContentObjectAttribute_ezpage_block_type_{$attribute.id}_{$zone_id}">
+    {def $allowed_zones = ''}
     {foreach ezini( 'General', 'AllowedTypes', 'block.ini' ) as $type}
-        <option value="{$type}">{ezini( $type, 'Name', 'block.ini' )}</option>
+        {set $allowed_zones = ezini( $type, 'AllowedZones', 'block.ini' )}
```

Comment:
My legacy memory might serve me wrong, but I think this needs to be wrapped in a `ezini_hasvariable` check, to avoid ezini warnings. If not set, variable will need to be to set on each iteration as well.